app/src/main/java/vn/com/tma/idlesmart/fragment/AlertDialogFragment.java

```diff
@@ -43,9 +33,9 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
     public void onViewCreated(View view, Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
         getDialog().requestWindowFeature(FEATURE_NO_TITLE);
-        String faultMessage = getArguments().getString("faultMessage");
-        String faultDesc = getArguments().getString("faultDesc");
-        final int faultId = getArguments().getInt("faultId");
+        String faultMessage = this.getArguments().getString("faultMessage");
+        String faultDesc = this.getArguments().getString("faultDesc");
+        final int faultId = this.getArguments().getInt("faultId");
```

Comment:
Should check if the arguments not existed, return the default value, i.e, `String` -> "", `int` -> 0